### ISSUE TITLE
Black and white title bar in Gnome Files using materia-compact

### ISSUE THREAD

**Issue by elParaguayo** [nana-4/materia-theme#290]
I've got an issue with Files in Gnome 3.30. Using Materia-Compact, the breadcrumb part of the title bar (in the centre) is light while the rest of the bar is dark. The white text makes it impossible to read.

![Inconsistent theme](http://i68.tinypic.com/20syjqq.png)

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

Required information:
- Materia version: 28 September 
- GTK+ 3 version: 3.24.1
- Distribution (and version): Arch Linux
- Desktop environment (and version): Gnome 3.30
- Related application (and version): Files 3.30.1-a3ca977a1
- Steps to reproduce: Open files.
- Screenshot(s): Above

**Comment by flipflop97** [comment]
This bug is also present with the non-compact light variant (and I think all other variants too), but only when the app window is large enough. When the app window is small the box around the breadcrumb bar doesn't appear.

I didn't have this issue a week ago so I think it may have to do with the Nautilus 3.30.1 update.

![screenshot from 2018-10-08 19-32-35](https://user-images.githubusercontent.com/13943260/46624425-04d98380-cb31-11e8-9810-44c451ee543a.png)  
*Large window - breadcrumb bar has a different background color and a border*

![screenshot from 2018-10-08 19-32-54](https://user-images.githubusercontent.com/13943260/46624427-06a34700-cb31-11e8-9ea7-0b11cc51ded0.png)  
*Small window - breadcrumb bar seems fine*

**Comment by ArchangeGabriel** [comment]
It definitively is Nautilus 3.30.1 that changed that. This is a dupe of: https://github.com/nana-4/materia-theme/issues/288. Read the GNOME thread linked there.

**Comment by elParaguayo** [comment]
Apologies. I did think that someone would have reported this already but didn't find this issue. I'll close this as a duplicate.

**Comment by actionless** [comment]
however i found out what nautilus is using for labels header button fg color instead of headerbar fg color:

(should be white as window control buttons, but it's black as on normal headerbar buttons)

![2018-10-09--1539112199_980x87_scrot](https://user-images.githubusercontent.com/1655669/46692482-b21ecb00-cc07-11e8-9b08-4172cf5cf571.png)

**Comment by actionless** [comment]
my bad, the main issue is reproducible only with maximized window

**Comment by actionless** [comment]
maximized issue could be fixed by adding:

```scss
headerbar {
  .background {
    background-color: $headerbar_bg;
  }
}
```

**Comment by actionless** [comment]
because if they will fix it in nautilus, some other developer could put box.background widget to the headerbar

**Comment by nana-4** [comment]
@elParaguayo No need to apologize. I closed #288 too early. Thanks for your report. :)

**Comment by nana-4** [comment]
@actionless For now I've added a workaround only to the Nautilus specific `.path-bar-box.background.frame`, but

> some other developer could put box.background widget to the headerbar in some other app

I hope it won't happen because it should be avoided, maybe 😥
